Modix/Modules/CoinModule.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Modix.Modules
+{
+    using System;
+    using System.Threading.Tasks;
+    using Discord.Commands;
+    using Modix.Utilities;
+
+    [Group("random"), Name("Coin"), Summary("A bunch of random commands"), ApplyChannelLimits]
+    public class CoinModule : ModuleBase
+    {
+        private readonly Random _r = new Random();
+
+        [Command("coin"), Summary("Flips a coin")]
+        public async Task FlipCoin()
+        {
+            var coin = _r.Next(0, 2);
+
+            if (coin == 0)
```

Comment:
If you wanted to go overkill have an `enum` and cast the result to said type and `.ToString()` the `enum`